### PR TITLE
🚸 Menu: Return focus to anchor after escape/enter keypress

### LINKED ISSUE #1704: Menu focus does not return on anchor element when closed
When `Menu` is closed when using keyboard navigation, the anchor element should be focused or next in line for focus according to https://www.w3.org/TR/wai-aria-practices-1.1/#menu

### PR DESCRIPTION
**Pull request by oddvernes**
Resolves #1704

### CODE REVIEW

File: libraries/core-react/src/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -64,6 +64,13 @@ const MenuContainer = forwardRef<HTMLDivElement, MenuContainerProps>(
     useGlobalKeyPress('Escape', () => {
       if (open && onClose !== null) {
         onClose()
+        anchorEl.focus()
+      }
+    })
+
+    useGlobalKeyPress('Enter', () => {
+      if (open && onClose !== null) {
+        if (window.document.contains(anchorEl)) anchorEl.focus()
```
Comment (mimarz) [inline libraries/core-react/src/components/Menu/Menu.tsx:73]: Should we also call the `onClose()` here?

**Review by mimarz — CHANGES_REQUESTED** [review]

File: libraries/core-react/src/components/Menu/Menu.tsx (hunk 1 above)
Comment (oddvernes) [reply libraries/core-react/src/components/Menu/Menu.tsx:73]: This is handled by MenuItem

Comment (mimarz) [reply libraries/core-react/src/components/Menu/Menu.tsx:73]: Ah, because its a `button` now right 🙈

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**